source/_integrations/matter.markdown

```diff
@@ -214,37 +214,24 @@ Use one of these methods if your Matter device was added to Apple Home or Google
       - Deselect the checkbox, then select **Submit**.
       - In the next step, provide the URL to your Matter server.
 
-### Share a device from Apple Home
-
-To allow Home Assistant to control the Matter device that has previously been added to Apple Home, follow these steps:
-
-1. In Apple Home find your device and select the cogwheel to view the **Accessory Settings**.
-2. Scroll all the way down and select **Turn On Pairing Mode**.
-3. You are now given a setup code, copy this to the clipboard.
-4. In the Home Assistant Companion App, go to {% my integrations title="**Settings** > **Devices & services**" %}.
-5. Select **Devices** > **Add device** and on the QR code scanner page, select **More options**.
-6. From the list, select your device and paste the code you just received from Apple Home.
-7. Once the device is connected with Home Assistant, the device page opens and you can control it from there.
-
-<lite-youtube videoid="nyGyZv90jnQ" videotitle="Share Matter device from Apple Home to Home Assistant"></lite-youtube>
-
-### Share a device from Google Home
+### To share a device from Apple or Google Home
```

Comment:
Maybe rephrase this a bit more, like:

To share a device from another Matter controller (like Apple or Google Home)